README.md

```diff
@@ -1,15 +1,16 @@
 ## **Disclaimer**
 
-This is not production ready or even a code complete project.
+This is not production ready nor feature complete. See [Issues](https://github.com/discentem/cavorite/issues) for upcoming features.
```

Comment:
nit: Might consider this a `work-in-progress tool` instead of `not production ready`...is any tool really production ready 😄